webrecorder/browsertrix
# feat: Localization workflow improvements

Partially addresses https://github.com/webrecorder/browsertrix/issues/1416



### Changes

- Extracts translatable text strings in pre-commit hook
- Updates ternary pluralization to use `pluralOf` instead
- Generates XLIFF for Spanish

### Manual testing

1. Regression test UI for number and plural formatting.
2. Verify staged changes to `frontend/src` will run localization string extraction on commit
